plugins/source/oracledb/resources/plugin/plugin_test.go

```diff
@@ -52,20 +53,24 @@ func createTable(ctx context.Context, db *sql.DB, table *schema.Table) error {
 		builder.WriteString(client.Identifier(column.Name))
 		builder.WriteString(" ")
 		builder.WriteString(client.SQLType(column.Type))
-		if column.CreationOptions.NotNull {
-			builder.WriteString(" NOT NULL")
-		}
-		if column.CreationOptions.Unique {
-			builder.WriteString(" UNIQUE")
-		}
-		if column.CreationOptions.PrimaryKey {
+		if column.PrimaryKey {
 			switch client.SQLType(column.Type) {
 			case "clob", "blob":
 			// nop, ORA-02329: column of datatype LOB cannot be unique or a primary key
 			default:
 				pk = append(pk, client.Identifier(column.Name))
 			}
+		} else {
+			// In OracleDB primary keys are implicitly NOT NULL and UNIQUE
+			// and it errors out if we try to do it explicitly
+			if column.NotNull {
+				builder.WriteString(" NOT NULL")
+			}
+			if column.Unique {
+				builder.WriteString(" UNIQUE")
+			}
 		}
+
```

Comment:
`continue` early for less nesting, nit
```suggestion
			continue
		}

		// In OracleDB primary keys are implicitly NOT NULL and UNIQUE
		// and it errors out if we try to do it explicitly
		if column.NotNull {
			builder.WriteString(" NOT NULL")
		}
		if column.Unique {
			builder.WriteString(" UNIQUE")
		}
```